sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -331,6 +323,14 @@ __SYCL_GenericCastToPtrExplicit_ToGlobal(const void *Ptr) noexcept {
           Ptr, __spv::StorageClass::CrossWorkgroup);
 }
 
+template <typename dataT>
+extern volatile __attribute__((opencl_global)) dataT *
```

Comment:
With built-ins added to clang, I wouldn't expect their declarations in headers. Could you please elaborate why there is both header and clang declarations now?

Reply:
These are prefixed with `__SYCL_` rather than `__spirv_`. We could easily just remove these and inline the body, but having this both handles the appropriate cast and the StorageClass argument, simplifying the calling code.

Reply:
Ah, sorry, I missed that the ones from headers have `__SYCL` prefixes.